mobile_catalogue_view/static/src/js/product_image_popup.js

```diff
@@ -0,0 +1,54 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { ProductCatalogKanbanRecord } from "@product/product_catalog/kanban_record";
+import { productCatalogKanbanView } from "@product/product_catalog/kanban_view";
+import { ProductCatalogKanbanRenderer } from "@product/product_catalog/kanban_renderer";
+import { useFileViewer } from "@web/core/file_viewer/file_viewer_hook";
```

Comment:
```suggestion
import { ProductCatalogKanbanRecord } from "@product/product_catalog/kanban_record";
import { productCatalogKanbanView } from "@product/product_catalog/kanban_view";
import { ProductCatalogKanbanRenderer } from "@product/product_catalog/kanban_renderer";

import { registry } from "@web/core/registry";
import { useFileViewer } from "@web/core/file_viewer/file_viewer_hook";
```

while adding import statements always remember to categorize/group them based on the module names